client/src/index.ts

```diff
@@ -1,31 +1,24 @@
 import { initFirebaseApp } from "./firebase";
 import { initApolloClient } from "./graphql";
-import { enableUserCookieUpdates } from "./modules/auth";
 
-export const init = (
-  firebaseOptions: Object,
-  graphqlUri: string,
-  autoUpdateUserCookie: boolean
-) => {
-  initFirebaseApp(firebaseOptions);
+export const init = async (firebaseOptions: Object, graphqlUri: string) => {
   initApolloClient(graphqlUri);
-
-  if (autoUpdateUserCookie) enableUserCookieUpdates();
+  await initFirebaseApp(firebaseOptions);
 };
 
 export { analytics } from "./firebase";
 
 export {
-  login,
+  loginEmailPass,
```

Comment:
Should I still export all the login/register functions from client package? Maybe I can use them only with `useSession` now?